lib/views/frontend/spree/checkout/payment/_stripe_additional_info.html.erb

```diff
@@ -0,0 +1,13 @@
+<%- if @order.has_checkout_step?('address') -%>
+  <script>
+    Spree.stripeAdditionalInfo = {
+      name: "<%= @order.bill_address.full_name %>",
+      address_line1: "<%= @order.bill_address.address1 %>",
+      address_line2: "<%= @order.bill_address.address2 %>",
+      address_city: "<%= @order.bill_address.city %>",
```

Comment:
This breaks LoD but we can fix it by passing `bill_address` to this partial

Reply:
fixing